src/layouts/default.vue

```diff
@@ -208,7 +208,7 @@ export default {
 
 <style scoped>
 .app {
-  @apply h-screen h-[100dvh];
+  @apply min-h-screen min-h-[100dvh];
```

Comment:
I'm still unsure how the priority of these two is set. They have the same specificity and target the same CSS property.

Reply:
If the device supports `dvh`, then the last property is used.
If it does not support `dvh`, then the last property is discarded, and h-screen (100vh) is used.

Reply:
But what if a device supports `dvh` but somehow the `min-h-screen` appears later and the device prioritises that?